be/src/exec/repeat_node.cpp

```diff
@@ -100,10 +93,18 @@ Status RepeatNode::get_repeated_batch(
                 continue;
             }
 
-            char* new_tuple = reinterpret_cast<char*>(dst_tuples[j]);
-            new_tuple += (*dst_it)->byte_size();
-            dst_tuples[j] = reinterpret_cast<Tuple*>(new_tuple);
-
+            if (dst_tuples[j] == nullptr) {
+                int size = row_batch->capacity() * (*dst_it)->byte_size();
```

Comment:
is it a typo? should we use row_batch's capacity to calculate one tuple's memory size?

Reply:
the memory of dst_tuples[j] is all the memory of jth tuple in the row batch